gradle.properties

```diff
@@ -14,9 +14,7 @@ bouncycastle.version=1.64
 rocksdb.version=6.5.3
 xodus.version=1.2.3
 # config
-jaxb-api.version=2.2.12
-jaxb-core.version=2.3.0.1
-jaxb-impl.version=2.3.2
+jaxb.version=2.3.3
```

Comment:
Did you check, if we can still load our extensions?

Reply:
This change is safe as we now use the same jakarta api and the same versions as our extensions.
Just verified that all open source extensions start normally.

Reply:
Nice, I guess that I am just a little bit jumpy after last time :)

Reply:
totally understandable